hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/SynthTxnUtils.java

```diff
@@ -35,8 +34,10 @@
  * Some utilities related to synthetic transaction bodies.
  */
 public class SynthTxnUtils {
-    public static final Key IMMUTABILITY_SENTINEL_KEY =
-            Key.newBuilder().keyList(KeyList.DEFAULT).build();
+    // This key can no longer be an empty key list as it fails validation.  Use this key for now
+    public static final Key IMMUTABILITY_SENTINEL_KEY = Key.newBuilder()
+            .contractID(ContractID.newBuilder().contractNum(1).build())
+            .build();
 
     public static final long THREE_MONTHS_IN_SECONDS = 7776000L;
```

Comment:
Is 90 days (=7776000) really 3 months?  I would have thought 3 months was 91+ days.  Is it too late to rename this to `NINETY_DAYS_IN_SECONDS`?

Reply:
I'm unfamiliar with the use of that constant.  It seems like a change can be addressed in another pr.